changes/461.doc.md

```diff
@@ -0,0 +1 @@
+doc: add to require system pip package on Linux distribution for backend.ai installation.
```

Comment:
Don't have to put `doc: ` prefix here because the file extension already define the section in the changelog as "Documentation Changes" according to `pyproject.toml`.